### ISSUE TITLE
Eliminate Trajectory Analysis code

### ISSUE THREAD

**Issue by kyleabeauchamp** [choderalab/yank#25]
So analyze.py has lots of trajectory analysis code that duplicates things I've already written in MDTraj.  We can replace a lot of this redundant code with only a few lines of MDTraj code.  

There is one remaining to-do item on the Repex side: a member function that slices the netCDF database and outputs MDTraj trajectories.  (See, e.g. https://github.com/choderalab/repex/issues/49).  

I think cleaning up this issue will lead to massive readability improvements and could make the Yank-Repex-MDTraj pipeline the go-to tool for analysis on repex datasets.

**Comment by jchodera** [comment]
We will ditch the trajectory analysis code now.

**Comment by Lnaden** [comment]
@andrrizzi are you still wanting to update analyze.py before the 1.0 release? If not we will want to move this to post 1.0

**Comment by jchodera** [comment]
We should definitely update `analyze.py` before the 1.0 release, at least minimally.

**Comment by Lnaden** [comment]
I would agree, but how much we think we can get done in the next week (release due 7 days) will determine which features on this list for analyze get done.

**Comment by andrrizzi** [comment]
Good point. Maybe we can just be very conservative with the features announced for analysis. We can always extend later, and probably we won't add a lot of fancy analysis routines before the 1.0 release.
